include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -890,4 +929,125 @@ operator()(
                 });
     }
 }
+
+template <
+        class ExecSpace,
+        class MemorySpace,
+        class BSplines,
+        class InterpolationDDim,
+        ddc::BoundCond BcLower,
+        ddc::BoundCond BcUpper,
+        SplineSolver Solver,
+        class... IDimX>
+template <class OutMemorySpace>
```

Comment:
```suggestion
template <class OutMemorySpace>
template <
        class ExecSpace,
        class MemorySpace,
        class BSplines,
        class InterpolationDDim,
        ddc::BoundCond BcLower,
        ddc::BoundCond BcUpper,
        SplineSolver Solver,
        class... IDimX>
```
seems to fix doxygen issue

Reply:
Thx, I did not know that

Reply:
I guess your fix passes doxygen requirement but does not compile ? I should revert it ?

Reply:
Ah ^^. I was hoping that was the merge.